### PR TITLE
Setup cosmos for release via unified pipelines.

### PR DESCRIPTION
**Pull request by mitchdenny**
Setup cosmos for release via unified pipelines.

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/cosmos/ci.yml [hunk 1]
```diff
@@ -7,14 +7,22 @@ resources:
...
     include:
     - master
+    - feature/*
+    - hotfix/*
+    - release/*
+    - restapi*
   paths:
     include:
     - sdk/cosmos/
+    - sdk/core/
```
Comment (weshaggard) [inline sdk/cosmos/ci.yml]: Why are we including core here?

Comment (mitchdenny) [reply sdk/cosmos/ci.yml]: As discussed - just for consistency.

**Pull request merged**